### PR TITLE
stubbed second 21 rules

### PR DESCRIPTION
**Pull request by brianpmaher**
2nd group of 21 stubs
- Added second stubs
- Changed all instanced of `INTEGER` to `INTEGER_LIT` and did the same for `FLOAT`, `FIXED`, and `STRING`

### CODE REVIEW

**Comment by brianpmaher** [comment]
Worth noting I did compile this

**Comment by seanrogers2657** [comment]
+1

File: parser.cs [hunk 1]
```diff
@@ -150,14 +150,14 @@ public class Parser {
 
     private void type(){
         switch(__lookahead.Type){
-            case TOKENS.INTEGER:
-                match(TOKENS.INTEGER);
```
Comment (JesseStephenBrown) [inline parser.cs:154]: Nice catch

Comment (JesseStephenBrown) [inline parser.cs:154]: On second thought, these should be INTEGER, FLOAT, STRING, etc because of rules 10-13. INTEGER_LIT is 023424, INTEGER is "Integer"

Comment (brianpmaher) [inline parser.cs:154]: Understood

**Comment by JesseStephenBrown** [comment]
+1

**Pull request merged**